doc/dev/development.rst

```diff
@@ -130,6 +123,38 @@ Our primary means of **communication** for development is the `PypeIt developers
 Slack <https://pypeit.slack.com>`_ and a biweekly telecon.  Contact `X
 Prochaska`_ for Slack access and/or the relevant Zoom link.
 
+
```

Comment:
I'm not sure where's most appropriate, but should we add something that states that it's a good idea to build and test the docs using sphinx to ensure the docstrings appear as you've intended?

Reply:
It's mentioned in the "Workflow" section.  I added a brief comment about checking the formatting of the docs after they're built.